Chargebee/Classes/Result/CBError.swift

```diff
@@ -29,6 +29,16 @@ extension CBError: LocalizedError {
             return errorResponse.message
         }
     }
+    public var httpStatusCode: Int {
+        switch self {
+        case .operationFailed(let errorResponse):
+            return errorResponse.httpStatusCode ?? 0
```

Comment:
Why are we using `0` as the fallback? 🤔

Reply:
Internally httpStatusCode is Optional value(Int? ). Just not to pass optional value to flutter(if i pass Optional value customer might ask why its Optional). Added default value of Int on the condition

Reply:
Can we change the param `httpStatusCode` in `CBErrorDetail` to not be optional?

Reply:
Yes, we can do that. let's check and update the PR.

Reply:
Done!